dvc/utils/html.py

```diff
@@ -20,9 +23,22 @@
 </script>"""
 
 
+class MissingPlaceholderError(DvcException):
+    def __init__(self, placeholder):
+        super().__init__(f"HTML template has to contain '{placeholder}'.")
+
+
 class HTML:
-    def __init__(self):
-        self.elements = []
+    PLACEHOLDER = "plot_divs"
```

Comment:
What about `{dvc_plots}`? But...

Reply:
From https://github.com/iterative/dvc.org/issues/2402#issuecomment-865831930:

> think about going with pure HTML and chaning the way the "templates" work - my idea would be to remove the requirement for the placeholder and just insert the plots after <body> tag.

Is this a better implementation? Cc @pared thanks

Reply:
I don't think removing the `PLACEHOLDER` is better. People may want to put other things in the `<body/>`, like branding, other kinds of images, or text.

Reply:
Good point.